### ISSUE TITLE
Bottom Sheet width in Material 3 doesn't allow beyond 640dp

### ISSUE THREAD

**Issue by dustinkerstein** [FlutterFlow/flutterflow-issues#5169]
### Can we access your project?

- [x] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When Material 3 is enabled, the Bottom Sheet component has a max width of 640dp (see Flutter Docs [here](https://api.flutter.dev/flutter/material/ModalBottomSheetRoute/constraints.html)). As noted in this [SO post](https://stackoverflow.com/a/78028381/6146641), as a workaround, passing `constraints: BoxConstraints.expand()` when `showModalBottomSheet()` is called allows the widget to take up 100% screen width.

### Expected Behavior

I'm currently manually adding the above bit of code after exporting, but it would be ideal if FlutterFlow exposed this as an option for when calling `showModalBottomSheet()` as an action:

![image](https://github.com/user-attachments/assets/ada59e54-6ffb-4717-94a0-17bb9b5d1373)

### Steps to Reproduce

1. Enable Material 3 design in FlutterFlow settings
2. Create a Bottom Sheet component with Width = 100%
3. Add an Action that shows this Bottom Sheet
4. Test on portrait phone device - note Bottom Sheet takes up 100% width
5. Test on landscape tablet device (min 640dp) and note Bottom Sheet doesn't take up 100% width

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITFTiMjfsItPvt9K+KrcYcFF+CcRM0McWOdEl+tuZwgbIoznBux3d8jCVFVvTtClaVNcPU2bjnwHw+iKjobpLcErNT+eGbZY+Mx2YQ3ke2iQb4yICrrhe39tO/tTIGqd5MKjnCJ4ANJaR04B12CULu6WcHPDFOu/Zwh9f6PHaOI=

### Visual documentation

See above

### Environment

```markdown
- FlutterFlow version: 5.0.18
- Platform: MacOS
- Browser name and version: NA
- Operating system and version affected: Android 14
```

**Comment by Alezanello** [comment]
You're absolutely right! I'll escalate this to the engineering team so they can work on a fix. Thank you for bringing this to our attention!

**Comment by jubox974** [comment]
Does we have any update about this @Alezanello  ?
